meshtastic/interfaces/ble/receive_service.py

```
@@ -0,0 +1,358 @@
+        now = time.monotonic()
+        with iface._state_lock:
+            iface._receive_recovery_attempts += 1
+            attempts = iface._receive_recovery_attempts
+            last_recovery = iface._last_recovery_time
+        if attempts > RECEIVE_RECOVERY_RAPID_FAILURE_THRESHOLD:
+            max_exponent = max(
+                int(
+                    math.floor(
+                        math.log2(max(RECEIVE_RECOVERY_MAX_BACKOFF_SEC, 1.0))
+                    )
+                ),
```

Ruff (0.15.5)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)